.github/CODEOWNERS

```
@@ -4,78 +4,79 @@
+/plugins/catalog-backend-module-bitbucket-cloud   @backstage/reviewers @backstage/catalog-core @pjungermann
+/plugins/catalog-backend-module-msgraph           @backstage/reviewers @backstage/catalog-core @pjungermann
+/plugins/catalog-graph                            @backstage/reviewers @backstage/catalog-core @backstage/sda-se-reviewers
+/plugins/circleci                                 @backstage/reviewers @adamdmharvey
+/plugins/cloudbuild                               @backstage/reviewers @trivago/ebarrios
+/plugins/code-coverage                            @backstage/reviewers @alde @nissayeva
+/plugins/code-coverage-backend                    @backstage/reviewers @alde @nissayeva
+/plugins/cost-insights                            @backstage/reviewers @backstage/silver-lining
+/plugins/cost-insights-*                          @backstage/reviewers @backstage/silver-lining
+/plugins/events-backend                           @backstage/reviewers @pjungermann
+/plugins/events-backend-module-aws-sqs            @backstage/reviewers @pjungermann
+/plugins/events-backend-module-azure              @backstage/reviewers @pjungermann
+/plugins/events-backend-module-bitbucket-cloud    @backstage/reviewers @pjungermann
+/plugins/events-backend-module-gerrit             @backstage/reviewers @pjungermann
+/plugins/events-backend-module-github             @backstage/reviewers @pjungermann
+/plugins/events-backend-module-gitlab             @backstage/reviewers @pjungermann
+/plugins/events-backend-test-utils                @backstage/reviewers @pjungermann
+/plugins/events-node                              @backstage/reviewers @pjungermann
+/plugins/explore                                  @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/explore-react                            @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/fossa                                    @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/gcalendar                                @backstage/reviewers @szubster @ptychu @kielosz @alexrybch
+/plugins/git-release-manager                      @backstage/reviewers @erikengervall
+/plugins/home                                     @backstage/reviewers @backstage/techdocs-core
+/plugins/ilert                                    @backstage/reviewers @yacut
+/plugins/jenkins                                  @backstage/reviewers @timja
+/plugins/jenkins-backend                          @backstage/reviewers @timja
+/plugins/kafka                                    @backstage/reviewers @nirga @andrewthauer
+/plugins/kafka-backend                            @backstage/reviewers @nirga @andrewthauer
+/plugins/kubernetes                               @backstage/reviewers @backstage/warpspeed
+/plugins/kubernetes-*                             @backstage/reviewers @backstage/warpspeed
+/plugins/microsoft-calendar                       @Abhay-soni-developer abhaysoni.developer@gmail.com
```

don't need your email here, it should be like this after you've rebased

```suggestion
/plugins/microsoft-calendar                       @backstage/maintainers  @Abhay-soni-developer
```